client/src/Questions/QuestionsForm.js

```diff
@@ -0,0 +1,71 @@
+import { useState, useEffect } from "react";
+import { useHistory, useParams } from "react-router-dom";
+import Api from "../../Api";
+
+const QuestionForm = () => {
+  const history = useHistory();
+  const { id } = useParams();
+  const [questions, setQuestions] = useState({
+    topic: "",
+    url: "",
```

Comment:
The model stores "question" and "answer", not "topic" and "url" (I think those are for Resource)